clippy.toml

```diff
@@ -1 +1,2 @@
 msrv = "1.76"
+ignore-interior-mutability = ["bytes::Bytes", "alloy_primitives::Bytes"]
```

Comment:
Where is this used? You should use `static` instead of `const` to silence this warning

Reply:
This shouldn't be allowed in clippy, please use Vec<u8> instead then

Reply:
Actually this is fine, bytes::Bytes is ignored by default as internal mutability is not exposed. Please add a note for this

Reply:
discussed internally and keeping this as this lint is a false positive for `bytes::Bytes` and therefore alloy wrapper as well

per clippy doc:
> It’s correct to use a struct that contains interior mutability as a key, when its implementation of Hash or Ord doesn’t access any of the interior mutable types. However, this lint is unable to recognize this, so it will often cause false positives in theses cases. The bytes crate is a great example of this.

> `ignore-interior-mutability`: A list of paths to types that should be treated like Arc, i.e. ignored but for the generic parameters for determining interior mutability (default: ["bytes::Bytes"])